src/vs/workbench/contrib/chat/browser/chatResponseAccessibleView.ts

```diff
@@ -66,16 +66,30 @@ class ChatResponseAccessibleProvider extends Disposable implements IAccessibleVi
 			responseContent = item.errorDetails.message;
 		}
 		if (isResponseVM(item)) {
-			const toolInvocation = item.response.value.find(item => item.kind === 'toolInvocation');
-			if (toolInvocation?.confirmationMessages) {
-				const title = toolInvocation.confirmationMessages.title;
-				const message = typeof toolInvocation.confirmationMessages.message === 'string' ? toolInvocation.confirmationMessages.message : stripIcons(renderMarkdownAsPlaintext(toolInvocation.confirmationMessages.message));
-				const terminalCommand = toolInvocation.toolSpecificData && 'command' in toolInvocation.toolSpecificData ? toolInvocation.toolSpecificData.command : undefined;
-				responseContent += `${title}`;
-				if (terminalCommand) {
-					responseContent += `: ${terminalCommand}`;
+			const toolInvocations = item.response.value.filter(item => item.kind === 'toolInvocation');
```

Comment:
If you have multiple interleaved sections of markdown and tool calls, this is pulling out the tool calls and putting them after all the markdown, couldn't that be confusing vs keeping the same order that we would see in the response?

Reply:
I think we should get feedback on this from users. We already alert when a tool requires user action and play a sound, so when a user opens the accessible view, they might be most interested in the actual response content - that they hear all at once otherwise.

Reply:
Also, based on my testing, the response after a tool has been invoked is more useful than reading the inputs that were used specifically.

Reply:
Or even a way to drill in or "expand" a section in the accessible view. "Used tool xyz (Press cmd+enter for more details)"

Reply:
There's no way to do that atm afaict. Let's start with this and get feedback?

Reply:
The first suggestion is just text

Reply:
We don't use the term tool in the rendered response, so don't think we want that here.